mo_net/__init__.py

```
@@ -1,3 +1,4 @@
+import os
```

Ruff (0.11.9)

`os` imported but unused

(F401)